docs/src/man/intro.md

```diff
@@ -83,3 +83,24 @@ There are five main components of Memento.jl that you can manipulate:
 5. IO
 
 The remainder of this manual will discuss how you can use these components to customize Memento to you particular application.
+
+## Using Memento from Julia Modules
+
+Some care needs to be taken when working with Memento from [precompiled modules](http://docs.julialang.org/en/latest/manual/modules/#module-initialization-and-precompilation).
+Specifically, it is important to note that if you want folks be able to configure your logger from outside the module you'll want to register the logger in your `__init__()` method.
+
+```julia
+__precompile__() # this module is safe to precompile
+module MyModule
+
+using Memento
+
+# Create our module level logger (this will get precompiled)
+const LOGGER = get_logger("MyModule")
```

Comment:
Should `@__MODULE__` be used in the example?

Reply:
On 0.6 you need to use `Compat.@__MODULE` so I figured this was the safer choice.

Reply:
Maybe add a comment that `@__MODULE__` is preferred on 0.7